### ISSUE TITLE
License for datasets

### ISSUE THREAD

**Issue by Li0ness** [szilard/benchm-ml#60]
Hi
Could you please help me to understand if MIT  covers datasets mentioned here
   Training datasets of sizes 10K, 100K, 1M, 10M are [generated](https://github.com/szilard/benchm-ml/blob/master/0-init/2-gendata.txt) 
   from the well-known airline dataset (using years 2005 and 2006). A test set of size 100K is generated from the same (using year 2007).
